showcase/BUILD.bazel

```diff
@@ -56,11 +58,14 @@ java_gapic_library(
     test_deps = [
         ":showcase_java_grpc",
 #        ":showcase_java_grpc_extended",
+         "@com_google_api_grpc_proto_google_common_protos//google/cloud/location:location_java_grpc",
     ],
     transport = "grpc+rest",
     deps = [
         ":showcase_java_proto",
         "@com_google_googleapis//google/api:api_java_proto",
+        "@com_google_api_grpc_proto_google_common_protos//:com_google_api_grpc_proto_google_common_protos",
```

Comment:
Same with this bazel dep

Reply:
Hm  I tried removing this dep but it appears to result in the following error:

```
(14:11:07) ERROR: /Users/mpeddada/IdeaProjects2/gapic-generator-java/showcase/BUILD.bazel:48:19: Building showcase/libshowcase_java_gapic.jar (1 source jar) failed: (Exit 1): java failed: error executing command external/remotejdk11_macos/bin/java -XX:-CompactStrings '--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED' '--add-exports=jdk.compiler/com.sun.tools.javac.main=ALL-UNNAMED' ... (remaining 17 arguments skipped)
bazel-out/darwin-fastbuild/bin/showcase/_javac/showcase_java_gapic/libshowcase_java_gapic_tmp/com/google/showcase/v1beta1/EchoClient.java:834: error: [strict] Using type com.google.cloud.location.ListLocationsRequest from an indirect dependency (TOOL_INFO: "@com_google_api_grpc_proto_google_common_protos//:com_google_api_grpc_proto_google_common_protos"). See command below **
  public final ListLocationsPagedResponse listLocations(ListLocationsRequest request) {
                                                        ^
 ** Please add the following dependencies: 
  @com_google_api_grpc_proto_google_common_protos//:com_google_api_grpc_proto_google_common_protos to //showcase:showcase_java_gapic 
 ** You can use the following buildozer command: 
buildozer 'add deps @com_google_api_grpc_proto_google_common_protos//:com_google_api_grpc_proto_google_common_protos' //showcase:showcase_java_gapic 

Target //showcase:update_gapic failed to build
Use --verbose_failures to see the command lines of failed build steps.
(14:11:07) INFO: Elapsed time: 138.978s, Critical Path: 36.58s
(14:11:07) INFO: 247 processes: 10 internal, 221 darwin-sandbox, 16 worker.
(14:11:07) FAILED: Build did NOT complete successfully
(14:11:07) ERROR: Build failed. Not running target
(14:11:07) FAILED: Build did NOT complete successfully

```

Reply:
Ah let me try this locally

Reply:
Tried it with:
```
java_gapic_library(
    name = "showcase_java_gapic",
    srcs = [":showcase_proto_with_info"],
    gapic_yaml = None,
    grpc_service_config = "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_grpc_service_config.json",
    # TODO(#1285): Enable rest_numeric_enums once https://github.com/googleapis/gapic-showcase/issues/1255 is
    # fixed.
    rest_numeric_enums = False,
    service_yaml = "@com_google_gapic_showcase//schema/google/showcase/v1beta1:showcase_v1beta1.yaml",
    test_deps = [
        ":showcase_java_grpc",
#        ":showcase_java_grpc_extended",
        "@com_google_googleapis//google/cloud/location:location_java_grpc",
    ],
    transport = "grpc+rest",
    deps = [
        ":showcase_java_proto",
        "@com_google_googleapis//google/api:api_java_proto",
        "@com_google_googleapis//google/cloud/location:location_java_proto",
   ],
)
```
and `mvn verify -P enable-integration-tests` works. I'm guessing it's that `@com_google_api_grpc_proto_google_common_protos//:com_google_api_grpc_proto_google_common_protos` and `"@com_google_googleapis//google/cloud/location:location_java_proto"` both contain the location dep?

Not sure which one is better...